index.bs

```diff
@@ -1623,10 +1627,11 @@ When a [=transient input source=] |source| for {{XRSession}} |session| begins it
 When a [=transient input source=] |source| for {{XRSession}} |session| ends its [=transient action=] the UA MUST run the following steps:
 
   1. Let |frame| be a new {{XRFrame}} with {{XRFrame/session}} |session| for the time the action occurred.
+  1. Let |x| and |y| be the screen coordinates of the touch event if |source| has {{XRInputSource/targetRayMode}} {{XRTargetRayMode/"screen"}}.
```

Comment:
Would we need to further define "screen coordinates" here?
* Is this in raw screen pixels or normalized coordinates?
* How do the coordinate axes react to screen rotation?
* How does (0, 0) react to browser chrome or other DOM elements that may take up part of the screen during an `"inline"` session?

If the goal is to allow interaction with WebGL-rendered screen-space UI, would it still be simplest for the app to raycast into their scene in 3D to avoid worrying about all of those 2D conventions that just cancel out in 3D?

Reply:
@bialpio what coordinates is Chrome using in the gamepad implementation currently?

My impression would be that this is in the rotated screen space, using screen px coordinates (like DOM APIs), and is relative to the top left of the rendered area.

Reply:
Would DOM pixel coordinates be useful for an app that is raycasting against WebGL-based screen-space UI? (the stated purpose of an app using `screenX`/`screenY` rather than DOM overlays for screen-space UI)

For an `"inline"` session, I can imagine an app doing its own normalization of `screenX`/`screenY` relative to their canvas width/height, so they can ultimately target scene geometry in their game engine's logical screen space.  However, for an `"immersive-ar"` session with no output canvas, does the app have enough information to reliably reason about the implicit "canvas size" that gives meaning to those coordinates?  Across UAs that make different decisions about fitting within system UI or notches on the edge of the screen, is there a predictable X and Y value that represents the right/bottom of the screen, so apps can do that normalization math?  And would any of that ever be more convenient than just raycasting with `targetRaySpace`?

Perhaps `screenX`/`screenY` are only usable reliably in `"inline"` sessions.  If so, should we spec them to be null in immersive sessions?

Reply:
Hmmm.. I was actually imagining this mostly to be useful for handheld AR, especially given that Chromium is exposing this via gamepad already (apparently at least one user wanted this -- we should find out why they needed it?)

I was was thinking this would be based on the output framebuffer size.